openedx/core/djangoapps/profile_images/views.py

```diff
@@ -15,10 +17,22 @@
     SessionAuthenticationAllowInactiveUser,
 )
 from openedx.core.lib.api.permissions import IsUserInUrl, IsUserInUrlOrStaff
-from openedx.core.djangoapps.user_api.accounts.image_helpers import set_has_profile_image, get_profile_image_names
+from openedx.core.djangoapps.user_api.accounts.image_helpers import (
+    get_profile_image_version,
+    set_profile_image_version,
+    get_profile_image_names,
+)
 from .images import validate_uploaded_image, create_profile_images, remove_profile_images, ImageValidationError
 
 
+def _make_image_version():
+    """
+    Generate a version based on hashing timestamp.  This is in a separate function
+    so its behavior can be overridden in tests.
+    """
+    return hashlib.md5(str(time.time())).hexdigest()[:8]
```

Comment:
What's the benefit of hashing the timestamp instead of using the full unique value? Furthermore, if that's what you want to do, then why cut this off at 8 chars? Does that compromise the uniqueness of this value?